projects/audit-logging.md

```diff
@@ -0,0 +1,196 @@
+- OTel collectors may lose audit logs in transit (i.e. no guarantee of delivery)
+
+See [Appendix A: Guarantee of Delivery] for more details
+
+### Goals, objectives, and requirements
+
+The goal of this project is to make OTel fit for audit logging purposes that meet compliance requirements of enterprise software providers, in particular:
+
+- REQ-CONV-01: Semantic conventions for application-level audit logs are defined
+- REQ-CONV-02: Semantic conventions for infrastructure-level audit logs are defined
+- REQ-APPL-01: Guaranteed delivery of audit logs exported via OpenTelemetry SDK
+- REQ-PIPE-01: OTel collector must provide guaranteed delivery of audit logs, including when its process is interrupted
+
+See [Appendix A: Guarantee of Delivery] for more details
+
+## Deliverables
+
+- semantic convention for audit logs
+- extend OTel APIs/SDKs for audit logging purposes (in collaboration with the respective SIG)
+- extend OTel collector for audit logging purposes (in collaboration with the respective SIG)
+
+## Staffing / Help Wanted
+
+The following vendors are interested in improving this area:
+- SAP (@mlenkeit, @FWinkler79)
+- Microsoft (@reyang)
+
+Other vendors are invited to join the discussion.
+
+### Required staffing
+
+* Project lead: SAP (name tbd)
+* Sponsors:
+  - @reyang
+  - tbd
+* GC liaison: @svrnm
+* Engineers:
+  * SAP will provide a prototype in two languages (tbd; likely two of Java, JavaScript, Go)
+* Maintainers/approvers: tbd
+
+## Timeline
+
+TBD based on community involvement.
+
+## Labels
+
+- audit-logging (tbc)
+
+## Project Board
+
+TODO: add link
+
+## SIG Meetings and Other Info
+
+TODO: add information
+
+## Appendix
+
+### Appendix A: Guarantee of Delivery
+
+In the context of this document, guarantee of delivery describes the ability of delivering audit logs from source to destination through OTel means while ensuring that all such signals arrive at the destination and/or providing the source with a means to handle failed delivery.
+
+Messaging protocols that support different levels of delivery guarantees may refer to this behavior as _at least once_ or _exactly once_, as opposed to _at most once_.
+
+We assume that every component that is involved in the delivery of audit logs from source to destination must support guarantee of delivery individually, rather than assuming that this ability can be provided by e.g. only the collector or SDK.
+
+The implications of guarantee of delivery can be illustrated with an example consisting of a workload, an OTel collector, and a durable storage. The workload acts as the source and produces audit logs via the OTel SDK/API. It writes the data via OTLP to the OTel collector. The OTel collector is configured to export audit logs to a durable storage that acts as the destination such as an S3 bucket.
```

Comment:
Minor, it is a bit weird to see "SDK/API", I think folks normally use "API/SDK".